### PR TITLE
#55 - Added Bb LTI MVP and Bb REST PoC directions

### PR DESCRIPTION
**Pull request by cbeach47**
First phase of the Bb LTI/REST directions.

### CODE REVIEW

File: guides/BlackboardLearnIntegrationGuide.md [hunk 1]
```diff
@@ -1,204 +1,137 @@
...
 
-The purpose of this guide is to provide system administrators with an understanding of the openEquella Blackboard integration process to enable successful integration. 
+The purpose of this guide is to provide system administrators with an understanding of the openEQUELLA / Blackboard integration process to enable successful integration. 
 
-Please note that this guide has been developed for Blackboard version 9.1 (SP16) and, as such, may differ in appearance to your own installation. 
+Please note this guide has been developed using Blackboard version v3800 and, as such, may differ in appearance to your own installation. 
+
+The openEQUELLA / Blackboard integration has historically been achieved with a building block and web service that was uploaded into the Blackboard site. This has presented some difficulties in the past and Blackboard retired their SOAP Web Services in Q2 2020 which effectively broke the openEQUELLA web service. Due to this upcoming loss of functionality, and the complexity of maintaining the building block and web service, a pure LTI / REST integration was developed.
```
Comment (ChristianMurphy) [inline guides/BlackboardLearnIntegrationGuide.md]:
```suggestion
The openEQUELLA / Blackboard integration has historically been achieved with a building block and web service that was uploaded into the Blackboard site. This has presented some difficulties in the past and Blackboard retired their [SOAP Web Services in Q2 2020](https://help.blackboard.com/node/28021) which effectively broke the openEQUELLA web service. Due to this upcoming loss of functionality, and the complexity of maintaining the building block and web service, a pure LTI / REST integration was developed.
```

File: guides/BlackboardLearnIntegrationGuide.md [hunk 2]
```diff
@@ -1,204 +1,137 @@
...
+Adopters will no longer be able to use the 'My Institution' integration that was part of the building block and web service integration.
 
 ## Prerequisites
 To successfully complete this installation the system administrator will require:
-* Sufficient privileges to access the Blackboard system integration functionality.
-* Sufficient privileges to access the openEquella User Management functionality 
+* Sufficient privileges to access the Blackboard system integration functionality
+* Sufficient privileges to access the openEQUELLA user management, LTI Consumers, and External System Connector functionality 
+
+In order to make REST calls into a Blackboard site, adopters will need to obtain a REST Application ID via https://developer.blackboard.com/ . This REST Application ID will then be used (via the same site), to create a REST Application key and secret that is tied to your Blackboard site URL. You will need this key and secret when configuring the integration in Blackboard. Please read this [post](https://community.blackboard.com/docs/DOC-4258-developer-groups-site-quotas-and-rate-limits) for more information on how Blackboard handles REST Application IDs.
```
Comment (ChristianMurphy) [inline guides/BlackboardLearnIntegrationGuide.md]:
```suggestion
In order to make REST calls into a Blackboard site, adopters will need to obtain a REST Application ID via <https://developer.blackboard.com>. This REST Application ID will then be used (via the same site), to create a REST Application key and secret that is tied to your Blackboard site URL. You will need this key and secret when configuring the integration in Blackboard. Please read this [post](https://community.blackboard.com/docs/DOC-4258-developer-groups-site-quotas-and-rate-limits) for more information on how Blackboard handles REST Application IDs.
```

File: guides/BlackboardLearnIntegrationGuide.md [hunk 3]
```diff
@@ -1,204 +1,137 @@
...
-4.  Check that the openEquella Plugin is available in Blackboard.
-5.  Check the configuration data in Blackboard.
-6. 
\ No newline at end of file
+### Configure an External System Connector
+1. Under **Settings > External System Connectors**, click on **Add new connector**
+2. Select **Blackboard REST**
+3. Configure a Connector name, and provide a Blackboard REST URL
+4. Click on **Test URL**
+5. Specify the REST API Key and REST API Secret from https://developer.blackboard.com/ .
```
Comment (ChristianMurphy) [inline guides/BlackboardLearnIntegrationGuide.md]:
```suggestion
5. Specify the REST API Key and REST API Secret from <https://developer.blackboard.com>.
```

**Review by ChristianMurphy — APPROVED** [review]

**Comment by cbeach47** [comment]
@ddelblanco / @cathfitz - FYI since the LTI portion and the migration B2 are released, I plan to merge this in tomorrow morning (US time).  Please let me know if there's any changes you'd like to see first.

**Comment by ddelblanco** [comment]
The documentation changes look very clear and exhaustive to me.  I feel they are a pretty good guide to help with the configuration of the LTI integration.

**Pull request merged**